tutorials/jms-output-exactlyonce/README.md

```diff
@@ -0,0 +1,62 @@
+## JMS ActiveMQ Output Example
+
+
+This application shows the use of JMS output operator by implementing AbstractJMSSinglePortOutputOperator.
```

Comment:
More than just an example of JMS Output operator: can we say something like this

"This application shows the use of JMS output operator and its exactly-once semantics by ...."